modules/ROOT/pages/7.3-release-notes.adoc

```diff
@@ -105,10 +105,16 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 {productname} {release-version} also includes the following bug fix<es>:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Notifications didn't position and resize properly when resizing the editor or toggling views.
+// #TINY-10894
 
-// CCFR here.
+Previously, the function responsible for recalculating the width of notifications only added a width if the notification's width exceeded the boundaries.
+
+As a consequence, when notifications were constrained by boundaries, an additional width was added. However, if the boundaries were later increased, the added width was not adjusted accordingly.
+
+{prouductname} {release-version} addresses this issue. Now, the function now not only adds width when the notification's width exceeds the boundaries but also removes this width when necessary.
```

Comment:
> **Now,** the function **now** not only
is this double `now` needed?

Reply:
```suggestion
{prouductname} {release-version} addresses this issue. Now, the function adds width when the notification's width exceeds the boundaries but also removes this width when necessary.
```

Reply:
@lorenzo-pomili Updated to remove second `now` as requested.